src/plugins/analysis/checksec/code/checksec.py

```diff
@@ -35,9 +36,8 @@ def process_object(self, file_object):
             else:
                 file_object.processed_analysis[self.NAME]['summary'] = []
         except (IndexError, json.JSONDecodeError, ValueError) as error:
-            file_object.processed_analysis[self.NAME]['summary'] = [
-                'Error - Firmware could not be processed properly: {}'.format(error)
-            ]
+            logging.warning(f'Error occurred during {self.NAME} analysis:', exc_info=True)
```

Comment:
Templating `self.NAME` doesn't seem sensible in the case of these messages since there is no generic-ness to the messages and you could add explicit-y by replacing it.